jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   verbose: false,
   transform: {
-    '^.+\\.(ts|tsx)$': 'ts-jest',
+    '^.+\\.(ts|tsx|js|jsx)$': 'ts-jest',
```

Comment:
Need to pass js files through typescript so importing js files in tests inside jaeger-ui works.